docs/content/6.overlays/6.notification.md

```diff
@@ -29,7 +29,7 @@ export default defineAppConfig({
   ui: {
     notifications: {
       // Show toasts at the top right of the screen
-      position: 'top-0 right-0'
+      position: 'top-0 right-0 bottom-auto'
```

Comment:
```suggestion
      position: 'top-0 bottom-auto'
```

Reply:
There is no need actually to redefine `right-0` as there is already `end-0` specified.